### ISSUE TITLE
There is no config for docker-compose

### ISSUE THREAD

**Issue by diimdeep** [Parisson/TimeSide#71]
I just tried to `docker-compose up` and get error 

```text
Can't find a suitable configuration file. Are you in the right directory?

Supported filenames: docker-compose.yml, docker-compose.yaml, fig.yml, fig.yaml
```

**Comment by thomasfillon** [comment]
Hi !
You should switch to the `dev` branch for now.

**Comment by yomguy** [comment]
Yes the docker composition will officially released in the master branch with 0.8 ;)

**Comment by thomasfillon** [comment]
@diimdeep 
We close this one but could reopen if needed.
